game/services.py

```diff
@@ -5,7 +5,7 @@
 from channels.layers import get_channel_layer
 
 from game import exceptions, serializers, models
-
+from .tasks import upgrade_building
 
 class GameSessionConsumerService:
```

Comment:
please, follow [PEP8](https://peps.python.org/pep-0008/#blank-lines)
```suggestion
from .tasks import upgrade_building


class GameSessionConsumerService:
```

Reply:
I would also remove `from .tasks import upgrade_building` and add `. tasks` to the line above